app/javascript/components/close_contact/CloseContact.js

```diff
@@ -33,14 +39,43 @@ class CloseContact extends React.Component {
   }
 
   toggleModal = () => {
-    let current = this.state.showModal;
-    this.setState({
-      showModal: !current,
-    });
+    let newState = {
+      showModal: !this.state.showModal,
+    };
+    if (this.state.showModal) {
+      // if we currently are showing the modal, that means they clicked cancel
+      // (because the rest of this method hasnt had time to fire, and change 'show' to false)
+      // When they click cancel, we want to null out all of the fields
+      newState = {
+        first_name: this.props.close_contact.first_name || '',
+        last_name: this.props.close_contact.last_name || '',
+        primary_telephone: this.props.close_contact.primary_telephone || '',
+        email: this.props.close_contact.email || '',
+        last_date_of_exposure: this.props.close_contact.last_date_of_exposure,
```

Comment:
can we add a `|| null` here similar to `enrolled_id`

Reply:
Yessir!